Get an error when use completion on zsh shell

Something like this: 

![demo](https://user-images.githubusercontent.com/14226737/162661009-01d230af-b6c8-46b4-a77f-27bd55fd4727.gif)

Here is a minimal repository for reproduce: [test-deno-cliffy](https://github.com/0x-jerry/test-deno-cliffy).

```ts
#!/usr/bin/env deno run --no-check

import {
  Command,
  StringType,
  CompletionsCommand,
} from 'https://deno.land/x/cliffy@v0.23.0/command/mod.ts'

class ScriptType extends StringType {
  complete() {
    return ['deno', 'cliffy']
  }
}

export const tt = new Command()
  .name('tt')
  .description('Run custom script in package.json, like yarn run.')
  .type('script', new ScriptType())
  .stopEarly()
  .arguments('<script:script> [...params:file]')
  .action((_, scriptName, params = []) => {
    console.log(scriptName, params)
  })

tt.version('v1.0.0').command('completions', new CompletionsCommand())

if (import.meta.main) {
  tt.parse()
}
```

Generated zsh completion

```sh
#!/usr/bin/env zsh
# zsh completion support for tt vv1.0.0

autoload -U is-at-least

# shellcheck disable=SC2154
(( $+functions[__tt_complete] )) ||
function __tt_complete {
  local name="$1"; shift
  local action="$1"; shift
  integer ret=1
  local -a values
  local expl lines
  _tags "$name"
  while _tags; do
    if _requested "$name"; then
      # shellcheck disable=SC2034
      lines="$(tt completions complete "${action}" "${@}")"
      values=("${(ps:\n:)lines}")
      if (( ${#values[@]} )); then
        while _next_label "$name" expl "$action"; do
          compadd -S '' "${expl[@]}" "${values[@]}"
        done
      fi
    fi
  done
}

# shellcheck disable=SC2154
(( $+functions[_tt] )) ||
function _tt() {
  local state

  function _commands() {
    local -a commands
    # shellcheck disable=SC2034
    commands=(
      'completions:Generate shell completions.'
    )
    _describe 'command' commands
    __tt_complete script script 
  }

  function _command_args() {
    case "${words[1]}" in
      completions) _tt_completions ;;
    esac
  }

  _arguments -w -s -S -C \
    '(- *)'{-h,--help}'[Show this help.]' \
    '(- *)'{-V,--version}'[Show the version number for this program.]' \
    '1: :_commands'\
    '2: :script-script'\
    '3:: :_files' \
    '*:: :->command_args'

  case "$state" in
    command_args) _command_args ;;
    script-script) __tt_complete script script  ;;
  esac
}

# shellcheck disable=SC2154
(( $+functions[_tt_completions] )) ||
function _tt_completions() {

  function _commands() {
    local -a commands
    # shellcheck disable=SC2034
    commands=(
      'bash:Generate shell completions for bash.'
      'fish:Generate shell completions for fish.'
      'zsh:Generate shell completions for zsh.'
    )
    _describe 'command' commands
  }

  function _command_args() {
    case "${words[1]}" in
      bash) _tt_completions_bash ;;
      fish) _tt_completions_fish ;;
      zsh) _tt_completions_zsh ;;
    esac
  }

  _arguments -w -s -S -C \
    '(- *)'{-h,--help}'[Show this help.]' \
    '1: :_commands' \
    '*:: :->command_args'

  case "$state" in
    command_args) _command_args ;;
  esac
}

# shellcheck disable=SC2154
(( $+functions[_tt_completions_bash] )) ||
function _tt_completions_bash() {

  _arguments -w -s -S -C \
    '(- *)'{-h,--help}'[Show this help.]'
}

# shellcheck disable=SC2154
(( $+functions[_tt_completions_fish] )) ||
function _tt_completions_fish() {

  _arguments -w -s -S -C \
    '(- *)'{-h,--help}'[Show this help.]'
}

# shellcheck disable=SC2154
(( $+functions[_tt_completions_zsh] )) ||
function _tt_completions_zsh() {

  _arguments -w -s -S -C \
    '(- *)'{-h,--help}'[Show this help.]'
}

# _tt "${@}"

compdef _tt tt
```

--- a/command/completions/_zsh_completions_generator.ts
+++ b/command/completions/_zsh_completions_generator.ts
@@ -195,7 +195,7 @@ function _${replaceSpecialChars(path)}() {` +
           const completionsPath: string = path.split(" ").slice(1).join(" ");
           const action = this.addAction(arg, completionsPath);
           args.push(
-            `${++argIndex}${arg.optionalValue ? "::" : ":"} :${action.name}`,
+            `${++argIndex}${arg.optionalValue ? "::" : ":"} :->${action.name}`,
           );
         }
       }

--- a/command/test/integration/fixtures/completions_generate_zsh.out
+++ b/command/test/integration/fixtures/completions_generate_zsh.out
@@ -112,7 +112,7 @@ function _completions_test_foo_help() {
     '(- *)'{-h,--help}'[Show this help.]' \
     '(-h --help -g --global)'{-g,--global}'[Foo option.]:val:->val-boolean' \
     '1: :_commands'\
-    '2:: :command-command'
+    '2:: :->command-command'
 
   case "$state" in
     val-boolean) __completions_test_complete val boolean foo help ;;
@@ -164,7 +164,7 @@ function _completions_test_foo_bar_help() {
     '(- *)'{-h,--help}'[Show this help.]' \
     '(-h --help -g --global)'{-g,--global}'[Foo option.]:val:->val-boolean' \
     '1: :_commands'\
-    '2:: :command-command'
+    '2:: :->command-command'
 
   case "$state" in
     val-boolean) __completions_test_complete val boolean foo bar help ;;
@@ -184,7 +184,7 @@ function _completions_test_help() {
     '(- *)'{-h,--help}'[Show this help.]' \
     '(-h --help -g --global)'{-g,--global}'[Foo option.]:val:->val-boolean' \
     '1: :_commands'\
-    '2:: :command-command'
+    '2:: :->command-command'
 
   case "$state" in
     val-boolean) __completions_test_complete val boolean help ;;
@@ -241,7 +241,7 @@ function _completions_test_completions_help() {
     '(- *)'{-h,--help}'[Show this help.]' \
     '(-h --help -g --global)'{-g,--global}'[Foo option.]:val:->val-boolean' \
     '1: :_commands'\
-    '2:: :command-command'
+    '2:: :->command-command'
 
   case "$state" in
     val-boolean) __completions_test_complete val boolean completions help ;;
@@ -292,7 +292,7 @@ function _completions_test_completions_bash_help() {
     '(- *)'{-h,--help}'[Show this help.]' \
     '(-h --help -g --global)'{-g,--global}'[Foo option.]:val:->val-boolean' \
     '1: :_commands'\
-    '2:: :command-command'
+    '2:: :->command-command'
 
   case "$state" in
     val-boolean) __completions_test_complete val boolean completions bash help ;;
@@ -343,7 +343,7 @@ function _completions_test_completions_fish_help() {
     '(- *)'{-h,--help}'[Show this help.]' \
     '(-h --help -g --global)'{-g,--global}'[Foo option.]:val:->val-boolean' \
     '1: :_commands'\
-    '2:: :command-command'
+    '2:: :->command-command'
 
   case "$state" in
     val-boolean) __completions_test_complete val boolean completions fish help ;;
@@ -394,7 +394,7 @@ function _completions_test_completions_zsh_help() {
     '(- *)'{-h,--help}'[Show this help.]' \
     '(-h --help -g --global)'{-g,--global}'[Foo option.]:val:->val-boolean' \
     '1: :_commands'\
-    '2:: :command-command'
+    '2:: :->command-command'
 
   case "$state" in
     val-boolean) __completions_test_complete val boolean completions zsh help ;;

